### PR TITLE
Results Page by Console

### LINKED ISSUE #22: Sort Result Cards by Console
An idea for how this is done is pictured below, but sorting the results as tabs is also good.

screen shot 2018-10-09 at 12 17 17 am

### PR DESCRIPTION
**Pull request by Feeshay**
Resolves #22 
Takes a dictionary of games with the game console as the key and separates the games based on console. Tabs are added depending on how many game consoles are in the dictionary. You can switch tabs.

Example of tab-switching:
image
image

### CODE REVIEW

**Review by hanarimawi — CHANGES_REQUESTED** [review]
Make toggle function arrow function instead of bind

**Review by hanarimawi — APPROVED** [review]

File: frontend/src/components/Results.js [hunk 1]
```diff
@@ -2,17 +2,70 @@ import React, { Component } from 'react'
...
+  constructor(props) {
+    super(props)
+
+    this.toggle = this.toggle.bind(this)
+    this.state = {
+      activeTab: '1'
+    }
+  }
+
+  toggle(tab) {
```
Comment (angadgarg25) [inline frontend/src/components/Results.js]: If you change this to `toggle = tab => {...}` you don't need to do the bind in the constructor, which makes the code cleaner.

**Review by caiyingshen — APPROVED** [review]
looks good!!

**Review by angadgarg25 — APPROVED** [review]
Looks good! :)

**Pull request merged**